modules/nvidia_plugin/wheel/setup.py

```diff
@@ -246,16 +256,38 @@ def clone_openvino_src(self):
                     cwd=self.openvino_src_dir,
                     on_fail_msg='Failed to update the OpenVINO git submodules')
 
+    def openvino_conan_install(self):
+        if not os.path.isdir(self.openvino_build_dir):
+            self.mkpath(self.openvino_build_dir)
+
+        run_command(["conan",
+                     "install",
+                     f'-of={self.openvino_build_dir}',
+                     '--build=missing',
+                     self.openvino_src_dir],
+                     cwd=self.openvino_build_dir,
+                     on_fail_msg='Failed to install conan dependecies for OpenVINO CMake Project')
+
     def configure_openvino_cmake(self):
         if not os.path.isdir(self.openvino_build_dir):
             self.mkpath(self.openvino_build_dir)
 
-        configure_command = [self.cmake_exec, f'-S{self.openvino_src_dir}', f'-B{self.openvino_build_dir}',
-                             f'-DCMAKE_BUILD_TYPE={self.build_configuration_name}',
+        python_include_dir = sysconfig.get_path("include")
+
+        configure_command = [self.cmake_exec,
+                             '-G', 'Unix Makefiles',
+                             f'-S{self.openvino_src_dir}',
+                             f'-B{self.openvino_build_dir}',
+                             '-DVERBOSE_BUILD=ON',
+                             '-DENABLE_NVIDIA=ON',
                              '-DENABLE_PYTHON=ON',
                              f'-DPYTHON_EXECUTABLE={sys.executable}',
-                             f'-DWHEEL_VERSION={WHEEL_VERSION}',
-                             '-DENABLE_WHEEL=ON']
+                             f'-DPYTHON_INCLUDE_DIR={python_include_dir}',
+                             '-DNGRAPH_PYTHON_BUILD_ENABLE=ON',
```

Comment:
- `NGRAPH_PYTHON_BUILD_ENABLE` is not available anymore
- `PYTHON_EXECUTABLE` and similar are replaced with `Python3_EXECUTABLE`, see https://github.com/openvinotoolkit/openvino/pull/19847
- `VERBOSE_BUILD` is not available anymore, https://cmake.org/cmake/help/latest/variable/CMAKE_VERBOSE_MAKEFILE.html should be used
- `ENABLE_NVIDIA` -> `BUILD_nvidia_plugin` ? But anyway, it's not needed since you passed NVIDIA plugin explicitly via `OPENVINO_EXTRA_MODULES`
- Why is cmake generator explicitly specified? It can be autodetected

Reply:
@ilya-lavrenov 
> `PYTHON_EXECUTABLE` and similar are replaced with `Python3_EXECUTABLE`

Why `Python3_EXECUTABLE` and not `PYTHON3_EXECUTABLE` ??
I suggest in future to change from `Python3_EXECUTABLE` -> `PYTHON3_EXECUTABLE` for consistency reason

> Why is cmake generator explicitly specified? It can be autodetected

It is recommended to use `which` for detecting proper command path, see https://docs.python.org/3/library/subprocess.html#popen-constructor

Reply:
@ilya-lavrenov What is current alternative to `plugins.xml` ?

Reply:
> I suggest in future to change from Python3_EXECUTABLE -> PYTHON3_EXECUTABLE for consistency reason

You can make this proposal to cmake 😄 https://cmake.org/cmake/help/latest/module/FindPython3.html

> It is recommended to use which for detecting proper command path, see

I meant `-G` option lists makefiles in your case. What if make command is missed, while ninja-build is installed?

> How external plugin could register currently ?

As I see you compile NVIDIA plugin with main OpenVINO, which means NVIDIA plugin is not external and plugins.xml is not required.
When plugin is built on top of prebuilt OpenVINO, them plugins.xml is auto-generated and plugin is responsible for its installation.

Reply:
@ilya-lavrenov 
> > I suggest in future to change from Python3_EXECUTABLE -> PYTHON3_EXECUTABLE for consistency reason
> 
> You can make this proposal to cmake 😄 https://cmake.org/cmake/help/latest/module/FindPython3.html

Yeah, I thought it is internal OpenVINO option :) Just confused myself )

> 
> > It is recommended to use which for detecting proper command path, see
> 
> I meant `-G` option lists makefiles in your case. What if make command is missed, while ninja-build is installed?

It is related to commands passed to `subprocess`, which Python tries to find

> 
> > How external plugin could register currently ?
> 
> As I see you compile NVIDIA plugin with main OpenVINO, which means NVIDIA plugin is not external and plugins.xml is not required. When plugin is built on top of prebuilt OpenVINO, them plugins.xml is auto-generated and plugin is responsible for its installation.

Maybe I can compile it with `-DENABLE_PLUGINS_XML=ON` ?

Reply:
> It is related to commands passed to subprocess, which Python tries to find

I don't understand how it's connected with subprocesses and Python, maybe I'm missing something. I suppose just need to remove `-G` to allow cmake to find generator on the system automatically.

> Maybe I can compile it with -DENABLE_PLUGINS_XML=ON ?

If you are building whole OpenVINO in this setup.py, then yes - it can be a solution (just to fallback to old behavior)